src/System.Private.CoreLib/shared/System/Globalization/CalendarData.Windows.cs

```diff
@@ -340,6 +344,7 @@ private static unsafe bool CallEnumCalendarInfo(string localeName, CalendarId ca
             }
 
             // Now we have a list of data, fail if we didn't find anything.
+            Debug.Assert(context.strings != null); // TODO-NULLABLE: <ISSUE NUMBER HERE> we have already initialized this earlier but compiler complains
```

Comment:
please let me know if you know the issue number (I'll search meantime), if I can't find it I'll open a new one

Reply:
Do you have an isolated repro for what the issue is that you're seeing?  That would help a) determine whether there's an existing issue for it, and b) be necessary to open a new issue.

Reply:
seems like passing managed callback to native code is the issue which I guess makes sense. Will remove the TODO

Reply:
> seems like passing managed callback to native code is the issue which I guess makes sense

Yup.